src/components/balance/modals/ModalTransferAmount.vue

```diff
@@ -249,6 +254,30 @@ export default defineComponent({
       }
     }
 
+    const getPayload = async (method: SubmittableExtrinsic<'promise'>, nonce: u32): Promise<Uint8Array> => {
```

Comment:
having this function directly in the component will make it difficult to reuse client extrinsic payloads. I suggest separating this into a separate function so that it can be used in multiple components.